nullaway/build.gradle

```diff
@@ -31,7 +31,8 @@ dependencies {
     annotationProcessor deps.apt.autoValue
     compileOnly deps.apt.autoServiceAnnot
     annotationProcessor deps.apt.autoService
-    compileOnly deps.build.jsr305Annotations
+    // Using api following the guidance at https://jspecify.dev/docs/using#gradle
+    api deps.build.jspecify
```

Comment:
@cpovirk this is encouraged, correct?  This will bring in the JSpecify jar onto the annotation processor path for users, but they will have it there anyway I think if they are using a recent version of Error Prone.  I think this could only cause a problem if for some reason they already have a dep on an old version of the JSpecify jar.

Reply:
I was wondering about Error Prone, too. All I've found there so far is a _test_ dep, but it could have missed a full dep, especially since I didn't look at all at _indirect_ deps. (So someday Error Prone will get it through Guava or Caffeine or whatever.)

I think that including it should be fine. I think you're right that there could in theory be trouble if some other annotation processor wanted the old package, but that seems very unlikely, and things would generally work out OK even in that case.

Reply:
Error Prone will have the dep in 2.30.0 (or 2.29.3 or whatever the next release after 2.29.2 is):
- https://github.com/google/error-prone/pull/4500
- https://github.com/google/error-prone/commit/bfe69f8afb06113e103a161132189daf0a67844c